provider/scripts/spark/offline_store_spark_runner.py

```diff
@@ -80,7 +80,7 @@ def execute_sql_query(job_type, output_uri, sql_query, spark_configs, source_lis
 
         dt = datetime.now()
         safe_datetime = dt.strftime("%Y-%m-%d-%H-%M-%S-%f")
-        output_uri_with_timestamp = f'{output_uri}{safe_datetime}/'
+        output_uri_with_timestamp = f"{output_uri}{safe_datetime}" if output_uri[-1] == "/" else f"{output_uri}/{safe_datetime}"
```

Comment:
Whats this do?

Reply:
basically, if the output location doesn't end with `/`, it adds so date time will be separate directory.

Reply:
Make this two different lines:
one line should add a / at the end, call it output_uri_with_trailing_slash
Then just add the safe_datetime to that

Reply:
could rewrite it as this too:
```suggestion
        output_uri_with_timestamp = f"{output_uri.rstrip('/')}/{safe_datetime}""
```